crates/cli/src/bin/wasm-bindgen-test-runner/main.rs

```diff
@@ -197,9 +197,17 @@ fn main() -> anyhow::Result<()> {
         .unwrap_or(20);
 
     if debug {
-        println!("Set timeout to {} seconds...", timeout);
+        println!("Set timeout to {} seconds...", test_timeout);
     }
 
+    let driver_timeout = env::var("WASM_BINDGEN_DRIVER_TIMEOUT")
+        .map(|timeout| {
+            timeout
+                .parse()
+                .expect("Could not parse 'WASM_BINDGEN_DRIVER_TIMEOUT'")
+        })
+        .unwrap_or(5);
```

Comment:
Maybe the default could also be raised if it causes timeouts? Maybe 10 or 20 seconds.

I mean, the worst that can happen by raising it is that tests take a bit longer to time out.